ecs-logs-collector.sh

```diff
@@ -167,12 +173,17 @@ collect_brief() {
   get_docker_logs
 }
 
-collect_debug() {
-  collect_brief
+enable_debug() {
+  init
   enable_docker_debug
   enable_ecs_agent_debug
 }
 
+collect_debug() {
```

Comment:
This function is kinda misleading now since it doesn't actually collect debug logs. I think I'd vote for eliminating it altogether an just calling `collect_brief` and `enable_debug` in the `debug` case on line 538.

Reply:
Makes sense. collect_debug() was the existing function name so I had left it that way, but removing it in favour of combining the collect and debug in the case statement does make it cleaner.